.github/workflows/builds.yml

```diff
@@ -340,27 +340,28 @@ jobs:
         env:
           cache-name: cache-electron-modules
         with:
-          key: ${{ runner.os }}-${{ env.cache-name }}-${{ secrets.CACHE_VERSION }}-${{ hashFiles('./package-lock.json') }}
+          key: ${{ runner.os }}-${{ env.cache-name }}-${{ secrets.CACHE_VERSION }}-${{ hashFiles('./pnpm-lock.yaml') }}
           path: ${{ env.ELECTRON_CACHE }}
       - name: Cache electron-builder modules
         uses: actions/cache@v3
         env:
           cache-name: cache-electron-builder-modules
         with:
-          key: ${{ runner.os }}-${{ env.cache-name }}-${{ secrets.CACHE_VERSION }}-${{ hashFiles('./package-lock.json') }}
+          key: ${{ runner.os }}-${{ env.cache-name }}-${{ secrets.CACHE_VERSION }}-${{ hashFiles('./pnpm-lock.yaml') }}
           path: ${{ env.ELECTRON_BUILDER_CACHE }}
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2
       - name: Use Node.js specified in the '.nvmrc' file
         uses: actions/setup-node@v3
         with:
-          node-version-file: ".nvmrc"
-      - name: Install pnpm
```

Comment:
If we remove these 2 lines, then will there be a possibility that we run with different version of pnpm?